watertap/unit_models/ultraviolet.py

```diff
@@ -223,13 +224,24 @@ def build(self):
             doc="Inactivation rate coeff.",
         )
 
+        self.rate_constant = Var(
```

Comment:
Just for my understanding- what is the use for the rate_constant in the model? I see it relates to UV intensity and inactivation rate but unsure of its significance beyond that.

Reply:
The `rate_constant` can be an alternative input for users instead of `inactivation_rate`. It is easier for users to get the `rate_constant` from the dose response curve with respect to exposure time. I see different formulations of the dose response model, someone provides the `rate_constant` and `exposure_time`, and someone provides `inactivation_rate` and `uv_dose`, so it depends. The `rate_constant` can also be obtained by the kinetic reactions (though haven't considered yet). And, another point is that there is an alternative EEO relationship with respect to the `rate_constant`, I didn't include it since it is just derived from substituting the `rate_constant` into the dose response model and require some unit conversions.

Reply:
Thanks for the nice explanation!